### PR TITLE
Add flag to csv-merger to pass the test images NVRs

### PR DESCRIPTION
**Pull request by aaguirrre**
We need a new flag in the csv-merger to be able to pass the test images NVRs as a parameter when creating the CSV

**What this PR does / why we need it**:
We are in the process of creating test images for some kubevirt components. These images contain test code only meant to be used internally for testing purposes. Right now the pull urls of these images appears in the CSV relatedImages field, and we decided it's preferrable to only have the NVR of these images in an annotation. Therefore we want to create this new flag in the csv-merger, in order to pass the test images NVRs and create an annotation with them.

**Reviewer Checklist**

> Reviewers are supposed to review the PR for every aspect below one by one. To check an item means the PR is either "OK" or "Not Applicable" in terms of that item. All items are supposed to be checked before merging a PR. 

- [ ] PR Message
- [ ] Commit Messages
- [ ] How to test
- [ ] Unit Tests
- [ ] Functional Tests
- [ ] User Documentation
- [ ] Developer Documentation
- [ ] Upgrade Scenario
- [ ] Uninstallation Scenario
- [ ] Backward Compatibility
- [ ] Troubleshooting Friendly

**Jira Ticket**:

```jira-ticket
NONE
```

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by nunnatsa** [comment]
Thanks for your PR. Please add to the PR description what is the nvr test image, and why do we need it in the csv.

File: tools/csv-merger/csv-merger.go [hunk 1]
```diff
@@ -333,6 +334,8 @@ func getHcoCsv() {
 
 	csvBase.Spec.RelatedImages = sortRelatedImages(csvBase.Spec.RelatedImages)
 
+	csvBase.ObjectMeta.Annotations["test-images-nvrs"] = *testImagesNVRs
```
Comment (tiraboschi) [inline tools/csv-merger/csv-merger.go:337]: we should avoid setting it when the CLI parameter is empty

**Review by tiraboschi — APPROVED** [review]

**Pull request merged**